step-capstone/src/components/AddItemButton.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import AddIcon from '@material-ui/icons/Add';
+import { Fab } from '@material-ui/core'
+import AddFormPopover from './AddFormPopover'
+
+export default function AddItemButton(props) {
+    const [anchorEl, setAnchorEl] = React.useState(null);
+
+    const handleClick = (event) => {
+        setAnchorEl(event.currentTarget);
+    };
+
+    const handleClose = () => {
```

Comment:
Does this mean that once the popover is open, it cannot be closed? If not, how can you close it?

Reply:
You can close it either by clicking out outside of it or either pressing cancel or save

Reply:
Another question out of curiosity, how did you implement the feature of "closing the popup by clicking outside of it"?

Reply:
I believe that simply the characteristic of the Popover component provided by Material UI: https://material-ui.com/components/popover/#popover

I didn't have to implement that part. The only thing I did have to do was tell it to close when pressing cancel or save